recipes/leftbar/general_row/general_row.vue

```diff
@@ -44,79 +44,82 @@
         </slot>
       </div>
-      <div
-        class="dt-leftbar-row__omega"
-      >
-        <dt-tooltip
-          v-if="dndText"
-          placement="top"
-          :message="dndTextTooltip"
-        >
-          <template #anchor>
-            <div
-              class="dt-leftbar-row__dnd"
-            >
-              {{ dndText }}
-            </div>
-          </template>
-        </dt-tooltip>
-        <div
-          v-if="activeVoiceChat"
-          class="dt-leftbar-row__active-voice"
-        >
-          <dt-icon
-            size="300"
-            name="activity"
-          />
-        </div>
-        <dt-tooltip
-          v-else-if="!!unreadCount && hasUnreads"
-          :message="unreadCountTooltip"
-          placement="top"
-        >
-          <template #anchor>
-            <dt-badge
-              kind="count"
-              type="bulletin"
-              data-qa="dt-leftbar-row-unread-badge"
-            >
-              {{ unreadCount }}
-            </dt-badge>
-          </template>
-        </dt-tooltip>
-      </div>
     </a>
     <div
-      v-if="hasCallButton"
-      class="dt-leftbar-row__action"
-      data-qa="dt-leftbar-row-action"
+      v-if="hasActions"
+      class="dt-leftbar-row__omega"
     >
       <dt-tooltip
-        :message="callButtonTooltip"
+        v-if="dndText"
         placement="top"
+        :message="dndTextTooltip"
       >
         <template #anchor>
-          <dt-button
-            class="dt-leftbar-row__action-button"
-            data-qa="dt-leftbar-row-action-call-button"
-            circle
-            size="xs"
-            kind="inverted"
-            :aria-label="callButtonTooltip"
-            @focus="actionFocused = true"
-            @blur="actionFocused = false"
-            @mouseleave="actionFocused = false"
-            @click.stop="$emit('call', $event)"
+          <div
+            class="dt-leftbar-row__dnd"
           >
-            <template #icon>
-              <dt-icon
-                name="phone"
-                size="200"
-              />
-            </template>
-          </dt-button>
+            {{ dndText }}
+          </div>
         </template>
       </dt-tooltip>
+      <div
+        v-if="activeVoiceChat"
+        class="dt-leftbar-row__active-voice"
+      >
+        <dt-icon
+          size="300"
+          name="activity"
+        />
+      </div>
+      <dt-tooltip
+        v-else-if="!!unreadCount && hasUnreads"
+        :message="unreadCountTooltip"
+        placement="top"
+      >
+        <template #anchor>
+          <dt-badge
+            kind="count"
+            type="bulletin"
+            data-qa="dt-leftbar-row-unread-badge"
+            class="dt-leftbar-row__unread-badge"
+          >
+            {{ unreadCount }}
+          </dt-badge>
+        </template>
+      </dt-tooltip>
+      <div
+        v-if="hasCallButton"
+        class="dt-leftbar-row__action"
+        data-qa="dt-leftbar-row-action"
+      >
```

Comment:
tabbing to the call button and then mousing over the button causes the badge to show up behind
![Screenshot 2023-06-15 at 3 04 31 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/171eda34-f7ce-4e03-b729-010f98cb9eb0)

Reply:
actually just realized this was a problem before this change as well... might just be the storybook example?

Reply:
I couldn't replicate

Reply:
It is the tooltip-container, seems like is offside


<img width="900" alt="image" src="https://github.com/dialpad/dialtone-vue/assets/89984179/82fb3834-1d2d-40a6-bfb9-db00e49533fe">

Reply:
What I can see 

`dt-leftbat-row--action-focused` is removed when you hover with the mouse.

I searched for it in the code and I found this handling:

```
@focus="actionFocused = true"
@blur="actionFocused = false"
@mouseleave="actionFocused = false"
```

I suspect it is an issue doing this way

Reply:
> actually just realized this was a problem before this change as well... might just be the storybook example?

This should be fixed now, removed `@mouseover` and `@mouseleave` events to prevent the mouse to remove the `dt-leftbar-row--action-focused`

Reply:
> It is the tooltip-container, seems like is offside
> 
> <img alt="image" width="900" src="https://user-images.githubusercontent.com/89984179/246427085-82fb3834-1d2d-40a6-bfb9-db00e49533fe.png">

You're watching the tooltip container of the button behind it, should be fixed now.

Reply:
> What I can see
> 
> `dt-leftbat-row--action-focused` is removed when you hover with the mouse.
> 
> I searched for it in the code and I found this handling:
> 
> ```
> @focus="actionFocused = true"
> @blur="actionFocused = false"
> @mouseleave="actionFocused = false"
> ```
> 
> I suspect it is an issue doing this way

Thanks! this helped me to fix the issue faster